Foundation/NSProcessInfo.swift

```diff
@@ -102,7 +102,17 @@ public class NSProcessInfo : NSObject {
     
     public var operatingSystemVersion: NSOperatingSystemVersion {
         get {
-            NSUnimplemented()
+            let osVersionDictionary = _CFCopySystemVersionDictionary().takeRetainedValue()._swiftObject as! [String: AnyObject]
+            guard let productVersion = osVersionDictionary[_kCFSystemVersionProductVersionKey._swiftObject] as? String else {
```

Comment:
the two calls here are doing implicit conversions between object types and struct types which will not work on linux